Missing headers for installing yay

As seen in my recording here https://asciinema.org/a/525125 we're lacking the stdlib.h for building the `yay` AUR helper, so until that's resolved, can't get an AUR helper going.

Fixed the multiple pacman.conf configs. Looks like I'm still getting this error when compiling:
```
In file included from /usr/include/errno.h:28,
                 from cgo-gcc-prolog:30:
/usr/include/bits/errno.h:26:11: fatal error: linux/errno.h: No such file or directory
   26 | # include <linux/errno.h>
      |           ^~~~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:113: yay] Error 2
```

Now just getting 
```
In file included from ../gopath/pkg/mod/github.com/!jguer/go-alpm/v2@v2.1.2/alpm.go:10:
/usr/include/alpm.h:42:10: fatal error: archive.h: No such file or directory
   42 | #include <archive.h>
      |          ^~~~~~~~~~~
compilation terminated.
make: *** [Makefile:113: yay] Error 2
==> ERROR: A failure occurred in build().
    Aborting...

```

--- a/README.md
+++ b/README.md
@@ -1,5 +1,5 @@
 # Deck Script For Working via Steam Deck
-This script will install some dev programs and temporarily disable the read-only filesystem so that you can work with terminal development tools. Note that this script was made from Desktop Mode on my personal Steam Deck plugged into a NexDock Touch.
+This script will install some dev programs and temporarily disable the read-only filesystem so that you can work with terminal development tools. Note that this script was made from Desktop Mode on my personal Steam Deck plugged into a NexDock Touch. Alongside installed programs to make the deck favorable to a dev/work environment when docked, this script may evolve over time to become interactive for other functionality as well.
 
 ## Usage
 First, let's open up the Konsole from the Steam Deck application menu at the bottom left of Desktop mode. Then paste these commands to run the script:

--- a/deckscript.sh
+++ b/deckscript.sh
@@ -4,11 +4,15 @@ echo ********************************
 echo ********************************
 echo "Now to install some things..." 
 ## First let's change that password
+echo ********************************
 echo "Please change your Deck admin password"
+echo ********************************
 passwd
 ## Install multilib stuff and disable readonly filesystem
-echo ****************************f****
+echo ********************************
 echo "Now we'll install and update terminal programs along with flatpak apps"
+echo ********************************
+sleep 2
 sudo steamos-readonly disable
 flatpak update --appstream
 flatpak update -y
@@ -19,15 +23,12 @@ sudo pacman-key --init
 sudo pacman-key --populate archlinux
 sudo locale-gen
 sudo pacman -Sy archlinux-keyring && pacman -Su
-echo '
-[multilib]
-Include = /etc/pacman.d/mirrorlist' | sudo tee --append /etc/pacman.conf
 sudo pacman --noconfirm -Syyu git go base-devel lib32-mesa vulkan-radeon lib32-vulkan-radeon vulkan-icd-loader lib32-vulkan-icd-loader neofetch vim cmake glibc
 ## oh yeah let's now add btop++ if we can
 wget -qO btop.tbz https://github.com/aristocratos/btop/releases/latest/download/btop-x86_64-linux-musl.tbz
 sudo tar xf btop.tbz -C /usr/local bin/btop
 ## install yay for an AUR helper, resolving header issues prior to install
 cd
-sudo pacman --noconfirm -S cmake pkg-config glibc
+sudo pacman --noconfirm -S cmake pkg-config glibc gcc libarchive linux-api-headers
 git clone https://aur.archlinux.org/yay.git && cd yay && makepkg -si
 ## should be done at this point, but might need furher tweaking

--- a/deckscript.sh
+++ b/deckscript.sh
@@ -13,6 +13,7 @@ sudo steamos-readonly disable
 flatpak update --appstream
 flatpak update -y
 flatpak install codium boxes lutris pupgui2
+flatpak install brave firefox
 ## pupgui2 is protonup-qt, also we may need to initialize pacman keys here as that helped on my deck
 sudo pacman-key --init
 sudo pacman-key --populate archlinux
@@ -20,9 +21,12 @@ sudo pacman -Sy archlinux-keyring && pacman -Su
 echo '
 [multilib]
 Include = /etc/pacman.d/mirrorlist' | sudo tee --append /etc/pacman.conf
-sudo pacman --noconfirm -Syyu git go base-devel lib32-mesa vulkan-radeon lib32-vulkan-radeon vulkan-icd-loader lib32-vulkan-icd-loader neofetch vim cmake
-flatpak install brave firefox
+sudo pacman --noconfirm -Syyu git go base-devel lib32-mesa vulkan-radeon lib32-vulkan-radeon vulkan-icd-loader lib32-vulkan-icd-loader neofetch vim cmake glibc
 ## oh yeah let's now add btop++ if we can
 wget -qO btop.tbz https://github.com/aristocratos/btop/releases/latest/download/btop-x86_64-linux-musl.tbz
 sudo tar xf btop.tbz -C /usr/local bin/btop
+## install yay for an AUR helper, resolving header issues prior to install
+cd
+sudo pacman --noconfirm -S cmake pkg-config glibc
+git clone https://aur.archlinux.org/yay.git && cd yay && makepkg -si
 ## should be done at this point, but might need furher tweaking

--- a/deckscript.sh
+++ b/deckscript.sh
@@ -17,6 +17,7 @@ flatpak install brave firefox
 ## pupgui2 is protonup-qt, also we may need to initialize pacman keys here as that helped on my deck
 sudo pacman-key --init
 sudo pacman-key --populate archlinux
+sudo locale-gen
 sudo pacman -Sy archlinux-keyring && pacman -Su
 echo '
 [multilib]

